README.md

```diff
@@ -13,12 +13,17 @@
 
 `avail-light` is a data availability light client that can do the following:
 
-* Listen for newly produced blocks.
+* Listen for newly mined blocks
 * Gain confidence for `N` *cells*, where *cell* is defined as a `{row, col}` pair.
-  > As soon as a new block is available, the light client attempts to gain confidence by asking for a proof from a full 
-  client via JSON-RPC.
-* Batch processces blocks in reverse order (i.e. prioritizes the latest blocks) for lower numbered blocks where no confidence 
-  is yet gained.
+  > As soon as a new block is available, the light client attempts to gain confidence by asking for a proof from a full client via JSON-RPC.
+* Batch processces blocks in reverse order (i.e. prioritizes the latest blocks) for lower numbered blocks 	where no confidence is yet gained.
+
+### Modes of Operation
+
+1. **Light client**: This is the basic mode of operation and is always active in whichever mode it works. If the APP_ID is not given then this mode will be commence. In this mode, the client on each header it receieves will do the random sampling using RPC calls. It gets random cells in return, which verifies and calculate the confidence.
```

Comment:
`will commence`
`gets proofs in return`
Some more proofreading required. Docs team can help here.